pkg/hubble/parser/seven/parser.go

```diff
@@ -395,11 +396,16 @@ func decodeLayer7(r *accesslog.LogRecord) *pb.Layer7 {
 			Type:   flowType,
 			Record: decodeHTTP(r.Type, r.HTTP),
 		}
-	case r.Kafka != nil:
+	case r.L7 != nil && r.L7.Proto == "kafka":
```

Comment:
minor: Should we extract "kafka" from here and L 511 to a constant?

Reply:
The most recent version got rid of the changes in hubble code, so this is not an issue any more :-)